src/monitor/version_compat.c

```diff
@@ -0,0 +1,62 @@
+/*-------------------------------------------------------------------------
+ *
+ * src/monitor/version_compat.h
+ *	  Compatibility macros for writing code agnostic to PostgreSQL versions
+ *
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the PostgreSQL License.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#if (PG_VERSION_NUM < 110000)
+
+/*
+ * The list_qsort API was introduced in Postgres 12:
+ *
+ * https://git.postgresql.org/gitweb/?p=postgresql.git;a=commit;h=ab72716778128fb63d54ac256adf7fe6820a1185
+ */
+
+#include "postgres.h"
+#include "nodes/pg_list.h"
+#include "version_compat.h"
+
+/*
```

Comment:
can we enclose this in #ifdef

Reply:
How do you mean?

Reply:
implementation is only necessary if we are below PG 11.  We can do something like
```c
#if (PG_VERSION_NUM < 110000)
/* implementation for list_qsort */
#endif 
```

Reply:
Well I applied that treatment to the whole file actually...

Reply:
I take this back. it was already there at top of the file